cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+    "    \n",
+    "    # Get improved consensus labels with label quality multiannotator using model pred probs\n",
+    "    label_quality_multiannotator = get_label_quality_multiannotator(multiannotator_labels, pred_probs, verbose=False)\n",
+    "    consensus_labels = label_quality_multiannotator[\"label_quality\"][\"consensus_label\"].tolist()\n",
+    "    \n",
+    "    unique_indices = len(indices_changed)\n",
+    "    indices_changed.update(list(np.where(consensus_labels != np.array(seen_consensus_labels[-1]))[0]))\n",
+    "    unique_indices = len(indices_changed) - unique_indices\n",
+    "    label_changes_from_prior = np.sum(consensus_labels != np.array(seen_consensus_labels[-1]))\n",
+    "    \n",
+    "    print(\"Num changes in consensus labels from previous itter: \", np.sum(consensus_labels != np.array(seen_consensus_labels[-1])))\n",
+    "    print(\"Num unique indices changed: \", unique_indices)\n",
+    "    # End to delete\n",
+    "    \n",
+    "    results = {\n",
+    "        \"pred_probs\": pred_probs,\n",
+    "        \"consensus_labels_in\": labels, # consensus labels used to train the model\n",
+    "        \"images\": images, \n",
+    "        \"consensus_labels_out\": consensus_labels, # new consensus labels generated from pred_probs\n",
+    "        \"label_changes_from_prior\": label_changes_from_prior, # num changes in consensus labels from previous itterations\n",
+    "        \"unique_indices_added\": unique_indices, # number of unique labels indices changed\n",
+    "    }\n",
+    "    \n",
+    "    model_results[itter] = results\n",
+    "    itter+=1\n",
+    "    \n",
+    "    if unique_indices == 0: # no more label improvement\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9aa26e05",
+   "metadata": {},
+   "source": [
+    "## 4. Measure consensus label accuracy and model performance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99e60410",
+   "metadata": {},
+   "source": [
+    "Since our annotators annotated a dataset `cifar10` to which there exist highly accurate `true_labels`, we can measure and report the accuracy of our methods against the ground truth. \n",
+    "\n",
+    "In a true multiannotator setting this would not be possible since ground truth labels will not exist. Instead we have [benchmarked](link paper or blogpost maybe) this method to ensure this method produces more accurate `consensus_labels` than common inustry methods for getting a consensus."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3758a5fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import and load ground truth labels (once again this is only for metrics, normally not have this information)\n",
+    "!cd $experiment_path && wget -nc 'https://cleanlab-public.s3.amazonaws.com/Multiannotator/cifar-10h/cifar-10h-worst25-coin20/c10h_test_labels.npy'\n",
```

Comment:
put all wget calls at the very beginning of notebook.

In any notebook that downloads data or installs packages, the first cell of the notebook should fetch data, the second cell should install packages, and the third cell should import depedencies.  

This way if things go wrong, the user doesn't only find out halfway through the notebook when it would be annoying to restart the kernel.

Reply:
Tried to separate true_labels from rest of the data as much as possible but this makes sense!